crates/sui-json-rpc/src/api/bls.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
to double check, files under api/ are just split results of api.rs right?

Reply:
Yes, this is to make it easier to maintain